msticpy/nbtools/nbwidgets.py

```diff
@@ -80,7 +80,7 @@ def __init__(
         id_vals: Optional[List[Any]] = None,
         val_attrs: Optional[List[str]] = None,
         nb_params: Optional[Dict[str, str]] = None,
-        ns: Dict[str, Any] = globals(),
+        ns: Dict[str, Any] = globals(),  # pylint: disable=invalid-name
```

Comment:
couldn't we quite easily re-name this variable. Something like 'name_space'?

Reply:
I've added this as comment in this issue #233